package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/inteli-api",
-  "version": "1.20.1",
+  "version": "1.20.2",
```

Comment:
please don't update package-locks by hand use `npm install`. You've missed a changeon ln 9 btw

Reply:
ok. though there was a better way